packages/opentelemetry-core/src/platform/node/sampling-probability.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Configured via user set env var: OTEL_SAMPLING_PROBABILITY.
+ * Defaults to 1.
+ * @returns sampling probability.
+ */
+export function getSamplingProbability(): number {
```

Comment:
Think we can use the same environment naming scheme and behavior here

Reply:
done, also cleaned up the history.